app/services/html/parser.rb

```diff
@@ -43,10 +42,8 @@ def prefix_all_images(width = 880, synchronous_detail_detection: false)
         # allow image to render as-is
         next if allowed_image_host?(src)
 
-        if synchronous_detail_detection && img
-          attribute_width, attribute_height = Articles::EnrichImageAttributes.image_width_height(img, TIMEOUT)
-          img["width"] = attribute_width
-          img["height"] = attribute_height
+        if synchronous_detail_detection
```

Comment:
The removal of the nil check on `img` gave me a bit of pause but I checked the code above and there is no way that nil check would fail on this line. Good call in removing it!